src/palette/internal/palettelayout.h

```diff
@@ -112,6 +135,10 @@ class PaletteLayout
 
         const engraving::MStyle& style() const;
         std::shared_ptr<engraving::IEngravingFont> engravingFont() const;
+        engraving::compat::DummyElement* dummyParent() const;
+
+        //! NOTE Temporarily, do not use
+        engraving::Score* donUseScore() const { return m_score; }
```

Comment:
It should probably be **dont**UseScore :)

Reply:
Thanks, fixed